index.html

```diff
@@ -358,6 +328,60 @@ <h2 class="footer-title">
         </ul>
         <a href="https://mail.google.com/" class="link-mail">ibrahimnaseer896.gmail.com</a>
     </footer>
+ 
+<!-- <div id="myModal" class="modal">
+
+  
+  <div class="modal-content">
```

Comment:
- [ ] OPTIONAL:  I can see lots of commented code here, if you do not need it anymore lets delete it and make sure that the project is clean and professional